03_launch_mgmt_cluster.sh

```diff
@@ -55,11 +55,11 @@ CAPM3_RUN_LOCAL="${CAPM3_RUN_LOCAL:-false}"
 
 if [ "${EPHEMERAL_CLUSTER}" == "kind" ]; then
   IRONIC_HOST="${PROVISIONING_URL_HOST}"
-  BMO_CONFIG="ironic-outside-config"
+  #BMO_CONFIG="ironic-outside-config"
```

Comment:
Why is this line commented out? If it is not needed remove it, if you have plan to restore it afterwards add a comment

Reply:
In case of kind, ironic will be runnning outside of the cluster and BMO is now deployed in CAPM3, so I think we can remove it or not sure TBH

Reply:
yes you can remove it

Reply:
Done.